reference/checkouts.v3.yml

```diff
@@ -400,9 +400,8 @@ paths:
                         name: Chambray Towel
                         url: 'http://store-url.mybigcommerce.com/all/chambray-towel/'
                         quantity: 1
-                        taxable: true
-                        image_url: >-
-                          https://cdn8.bigcommerce.com/s-{store_hash}/products/158/images/382/foglinenbeigestripetowel1b_1024x1024__83011__60806.1534344511.330.500.jpg?c=2
+                        is_taxable: true
```

Comment:
Can you please verify that `is_taxable` is correct? Per Jira and examples, this filed should be changed to `taxable`.